src/frontend/src/flows/confirmRedirect.ts

```diff
@@ -1,15 +1,51 @@
 import { html, render } from "lit-html";
 import { FrontendHostname } from "../../generated/internet_identity_types";
 
-const pageContent = (hostName: string) => html`
+// We show a tooltip giving more information about the "application-specific"
+// nature of pseudonyms. This is the text that gets displayed when hovering.
+const textTooltip = `Internet Identity issues a unique "pseudonym" for each application requesting
+authentication. Since this pseudonym is unique for each application you
+authenticate for, it cannot be used to track you.
```

Comment:
```suggestion
authenticate for, applications cannot use it to track you.
```

Suggestion from @Dfinity-Bjoern